README.md

```diff
@@ -1 +1,318 @@
-# sql-joins
+# SQL Joins
+
+## Objectives
+
+1. Define SQL joins and understand when and why to use them.
+2. Define the different types of SQL joins: inner, outer, left outer, right outer
+3. Practice writing join statements. 
+
+## What Is a JOIN?
+
+A SQL JOIN clause is a way to combine rows from two or more tables, based on a common column between them. The great thing about relational databases is that they are just that––*relational*. Relational databases allow us not only to store data that is interconnected, but to retrieve that data in ways that reflect that interconnectivity. 
+
+Let's say, for example, we have two tables, a Cats table and an Owners table. Cats and owners are associated by a foreign key of `owner_id` in the Cats table. How would we craft a query that would grab us all of the cats with a particular owner, and even include information about that owner in the data returned to us by that query? We know how to write a `SELECT` statement that gets us all of the cats with a particular `owner_id`. For example:
```

Comment:
This reading comes after the reading and code along on table relations w discussion of foreign keys https://github.com/learn-co-curriculum/table-relations